_posts/config/2017-05-05-network-configuration.md

```diff
@@ -44,7 +44,7 @@ If the network interface is _Enabled for LAN_ and manually configured (i.e., not
 The **DHCP & NAT** tab contains the following configuration parameters:
 
 - **Router Mode**
-  - DHCP and NAT - indicates that both DHCP server and NAT are enabled.
+  - DHCP and NAT - indicate that both DHCP server and NAT are enabled.
```

Comment:
This should not be changed, because word "indicates" is connected to the menu entry, which is "DHCP & NAT". 
It is like:
"option 1" - indicates...
"option 2" - indicates...

"option1 and option2 " can be "abc and bfer" or just "abc". Please change back.